extension/chrome/elements/shared/key_errors.ts

```diff
@@ -23,6 +23,7 @@ export class KeyErrors {
   ) {}
 
   public handlePrivateKeyError = async (exception: unknown, origPrv: Key, setupOptions?: SetupOptions) => {
+    console.log('test');
```

Comment:
just added and remove this temporarily to somehow fixed CI test not working correctly aftering clicking 'rerun' which doesn't produce the desired effect.